packages/dashboard/src/helpers/filterData.ts

```diff
@@ -0,0 +1,73 @@
+import { SharedFilter } from "../types/SharedFilter"
+import { generateValuesForFilter } from "./generateValuesForFilter"
+
+const findFilterTier = (filters: SharedFilter[], sharedFilter: SharedFilter) => {
+    if (!sharedFilter.parents?.length) {
+      return 1
+    } else {
+      let parent = filters.find(filter => sharedFilter.parents!.includes(filter.key))
+      if (!parent) return 1
+      return 1 + findFilterTier(filters, parent)
+    }
+  }
+
+export const filterData = (filters: SharedFilter[], _data: Object[], filterBehavior) => {

```

Comment:
Run prettier to resolve alignment issues. Prettier doesn't run when the `-n` flag is used, so its possible this was bypassed.

Reply:
I think prettier might not be working on ts files. let me look into it.

Reply:
I think this is fixed however I still had to run the no verify flag as it was throwing this error without it:
`Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Failed to load plugin 'flowtype' declared
in '--config » eslint-config-react-app': Package subpath './lib/rules/no-unused
-expressions' is not defined by "exports" in C:\Users\pro7\dev\cdc-open-viz\node
_modules\eslint\package.json
    at new NodeError (node:internal/errors:405:5)
    at exportsNotFound (node:internal/modules/esm/resolve:359:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:695:9)
    at resolveExports (node:internal/modules/cjs/loader:567:36)
    at Module._findPath (node:internal/modules/cjs/loader:636:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (C:\Users\pro7\dev\cdc-open-viz\node_modules\eslint-pl
ugin-flowtype\dist\rules\noUnusedExpressions.js:8:51)
husky > pre-commit hook failed (add --no-verify to bypass)`